lib/utils.py

```diff
@@ -284,10 +284,11 @@ def xlog_rows(xlog_path):
         Assume tarantool and tarantoolctl is in PATH.
     """
     cmd = ['tarantoolctl', 'cat', xlog_path, '--format=json', '--show-system']
+    process = None
     with open(os.devnull, 'w') as devnull:
         process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=devnull)
     for line in process.stdout.readlines():
-        yield json.loads(line)
+        yield json.loads(bytes_to_str(line))
```

Comment:
I would extract it from the 'test: add unit test' commit, because this change fixes the real problem. It worth to quote the documentation to make it clear that you solve the Python 3.5 problem here.

Reply:
moved to a separate commit

Reply:
I was stuck for a while trying to understand why everything works without the change (just because I have Python 3.9 installed on my system), so I still think that we should cite the json module documentation or mention how Python 3.5 and 3.6+ are different here in some other way. Sorry for nitpicking.

Reply:
Updated commit message:
```
    lib: use bytes_to_str() in xlog_rows()
    
    Since Python 3.6+ in json.loads() object that should be deserialized can
    be of type bytes or bytearray. The input encoding should be UTF-8,
    UTF-16 or UTF-32 [1]. Patch follows up
    commit 395edeb6b743c4479a62dd2183062124973d2b2a
    ('python3: decouple bytes and strings') and it is a part of
    task with switching to Python 3.
    
    1. https://docs.python.org/3/library/json.html
    
    Follows up: #20
```

Reply:
The important thing is that it cannot by bytes in Python 3.5 (it is the same as 'since Python 3.6+ is can', but the accent differs), but anyway now it is much easier to spot the reason of the change. Thanks.